oblt-cli/run/action.yml

```diff
@@ -24,7 +24,8 @@ runs:
         slack-channel: ${{ inputs.slack-channel }}
         username: ${{ inputs.username }}
     - name: run oblt-cli
-      run: oblt-cli --verbose ${{ inputs.command }}
+      run: oblt-cli --verbose ${COMMAND}
```

Comment:
It still allows shell injection. This action, by design, allows it. There is no way to fix it. The solution is to implement actions for each command we use and manage all arguments we use as inputs.